data/schema.rdfa

```diff
@@ -11328,6 +11329,64 @@ postponing for 1.6.
+      <span>Range: <a property="http://schema.org/rangeIncludes" href="http://schema.org/Organization">Organization</a></span>
+    </div>
+    <div typeof="rdf:Property" resource="http://schema.org/broadcastDisplayName">
+      <span class="h" property="rdfs:label">broadcastDisplayName</span>
+      <span property="rdfs:comment">The name displayed in the channel guide. For many US affiliates, it is the network name.</span>
+      <span>Domain: <a property="http://schema.org/domainIncludes" href="http://schema.org/TelevisionStation">TelevisionStation</a></span>
+      <span>Range: <a property="http://schema.org/rangeIncludes" href="http://schema.org/Text">Text</a></span>
+    </div>
+    <div typeof="rdf:Property" resource="http://schema.org/timezone">
+      <span class="h" property="rdfs:label">timezone</span>
+      <span property="rdfs:comment">The timezone in &lt;a href=&#39;http://en.wikipedia.org/wiki/ISO_8601&#39;&gt;ISO 8601 format&lt;/a&gt;for which the TV station bases its broadcasts.</span>
+      <span>Domain: <a property="http://schema.org/domainIncludes" href="http://schema.org/TelevisionStation">TelevisionStation</a></span>
+      <span>Range: <a property="http://schema.org/rangeIncludes" href="http://schema.org/Text">Text</a></span>
+    </div>
+    <div typeof="rdf:Property" resource="http://schema.org/isLiveBroadcast">
+      <span class="h" property="rdfs:label">isLiveBroadcast</span>
+      <span property="rdfs:comment">True is the broadcast is of a live event..</span>
+      <span>Domain: <a property="http://schema.org/domainIncludes" href="http://schema.org/BroadcastEvent">BroadcastEvent</a></span>
+      <span>Range: <a property="http://schema.org/rangeIncludes" href="http://schema.org/Boolean">Boolean</a></span>
+    </div>
+    <div typeof="rdfs:Class" resource="http://schema.org/TelevisionChannel">
+      <span class="h" property="rdfs:label">TelevisionChannel</span>
+      <span property="rdfs:comment">A unique instance of a TelevisionStation on a BroadcastService's lineup.</span>
+      <span>Subclass of: <a property="rdfs:subClassOf" href="http://schema.org/Intangible">Intangible</a></span>
+   </div>
+    <div typeof="rdf:Property" resource="http://schema.org/televisionChannelId">
+      <span class="h" property="rdfs:label">televisionChannelId</span>
+      <span property="rdfs:comment">The unique address by which the TV station can be identified in a provider lineup. In US, this is typically a number.</span>
+      <span>Domain: <a property="http://schema.org/domainIncludes" href="http://schema.org/TelevisionChannel">TelevisionChannel</a></span>
+      <span>Range: <a property="http://schema.org/rangeIncludes" href="http://schema.org/Text">Text</a></span>
+    </div>
+    <div typeof="rdf:Property" resource="http://schema.org/inBroadcastLineup">
+      <span class="h" property="rdfs:label">inLineup</span>
```

Comment:
Good catch. I updated the pull request with the new label.


Vicki Tardif Holland | Ontologist | vtardif@google.com


On Fri, Mar 20, 2015 at 2:35 PM, tmarshbing <notifications@github.com>
wrote:

> In data/schema.rdfa
> <https://github.com/schemaorg/schemaorg/pull/389#discussion_r26878404>:
>
> > +      <span>Domain: <a property="http://schema.org/domainIncludes" href="http://schema.org/BroadcastEvent">BroadcastEvent</a></span>
> > +      <span>Range: <a property="http://schema.org/rangeIncludes" href="http://schema.org/Boolean">Boolean</a></span>
> > +    </div>
> > +    <div typeof="rdfs:Class" resource="http://schema.org/TelevisionChannel">
> > +      <span class="h" property="rdfs:label">TelevisionChannel</span>
> > +      <span property="rdfs:comment">A unique instance of a TelevisionStation on a BroadcastService's lineup.</span>
> > +      <span>Subclass of: <a property="rdfs:subClassOf" href="http://schema.org/Intangible">Intangible</a></span>
> > +   </div>
> > +    <div typeof="rdf:Property" resource="http://schema.org/televisionChannelId">
> > +      <span class="h" property="rdfs:label">televisionChannelId</span>
> > +      <span property="rdfs:comment">The unique address by which the TV station can be identified in a provider lineup. In US, this is typically a number.</span>
> > +      <span>Domain: <a property="http://schema.org/domainIncludes" href="http://schema.org/TelevisionChannel">TelevisionChannel</a></span>
> > +      <span>Range: <a property="http://schema.org/rangeIncludes" href="http://schema.org/Text">Text</a></span>
> > +    </div>
> > +    <div typeof="rdf:Property" resource="http://schema.org/inBroadcastLineup">
> > +      <span class="h" property="rdfs:label">inLineup</span>
>
> Your label still says "inLineup"
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/schemaorg/schemaorg/pull/389/files#r26878404>.
>